### DISCUSSION TITLE
PDF files local - how?

### DISCUSSION THREAD

**Discussion by wall121** [TandoorRecipes/recipes#1379] [Q&A] [answered]
Hi,

I don't get it how to view PDF files. I justed installed Tandoor on a Synology following the great tutorial of @DiversityBug. Everything works fine, great application, nice design. Easy import recipes based on URLs

 But I don't find where I have to put existing PDF files on the local server and and how exactly I have to configure it  in the App

I can't change the "Speicher" name nor do I know where I have to put the PDF files locally / on the Synology.

![Tandoor1](https://user-images.githubusercontent.com/51135059/150205982-76bbcfbb-2c91-4182-be3b-2a1f7a31921c.png)

I tried to copy PDF recipes into the recipe folder but it doesn't show up.
![Tandoor2](https://user-images.githubusercontent.com/51135059/150207004-9e960a53-63b7-40e8-aa87-54097c00654c.png)

Any idea / help?

Thx

Wally

**Comment by TWART016** [comment, marked as answer]
Hi,

I think the `mediafiles/recipes` folder are for the images of recipes.

In the compose I mount a folder to the `web_recipes` container
`      - ./external_recipe:/opt/recipe/external_recipe`

In Tandoor you have to click on `Externe Speicherquellen verwalten` and create a new storage backend
![image](https://user-images.githubusercontent.com/29805669/150238673-43473df4-8e40-4a00-a3af-bdf9ba47e186.png)

**Reply by wall121** [reply]
Thx, that helps and works!
